packages/main/src/themes/RatingIndicator.css

```diff
@@ -0,0 +1,37 @@
+:host(:not([hidden])) {
+	display: inline-block;
+	font-size: 1.5rem;
+	cursor: pointer;
+}
+
+:host([disabled]),
+:host([read-only]) {
+	opacity: .8;
+	cursor: initial;
+	outline: none;
+}
+
+:host([size="Large"]) {
+	font-size: 2rem;
+}
+
+:host([_focused]) {
+	outline: 1px dotted var(--sapContent_FocusColor);
```

Comment:
The focus should be just around the stars:

<img width="193" alt="Screenshot 2020-06-05 at 10 48 54 AM" src="https://user-images.githubusercontent.com/15702139/83850849-2d58bd00-a71a-11ea-993d-126177cc090a.png">

currently is:

<img width="138" alt="Screenshot 2020-06-05 at 10 49 03 AM" src="https://user-images.githubusercontent.com/15702139/83850900-3ba6d900-a71a-11ea-9340-94ac96f65db9.png">

The problem is not with the css for the outline, but the stars itself or the wrapper.